pylint/checkers/refactoring.py

```diff
@@ -102,6 +103,10 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'weird bugs in your code. You should always use parentheses '
                   'explicitly for creating a tuple.',
                   {'minversion': (3, 0)}),
+        'R1708': ("The mutable sequence (%s) is modified within the loop",
```

Comment:
This is not a refactoring message, but rather a warning, so this should start with ``W`` instead.